include/odp/api/spec/dma_types.h

```diff
@@ -492,6 +499,50 @@ typedef struct odp_dma_transfer_param_t {
 	 */
 	odp_dma_seg_t *dst_seg;
 
+	/** Transfer options
+	 *
+	 *  Options to further configure the transfer.
+	 *  Initialize all unused bits to zero.
```

Comment:
odp_dma_transfer_param_init() is used to initialize the struct. So, this sentence could just define the default values:
"The default value for all option bits is zero."